.github/workflows/ci.yml

```diff
@@ -122,20 +122,6 @@ jobs:
       os: ${{ matrix.target }}
       changed: ${{ needs.changes.outputs.node-changes == 'true' }}
 
-  node-validation:
```

Comment:
I think we still need this in case there are something wrong with autofix.

Reply:
Do we still need to keep these?
https://github.com/rolldown/rolldown/blob/4c9feeb058f0212228e430450004967c99af9177/.github/workflows/ci.yml#L154-L158

Reply:
Format could be removed, since we have autofix.

Reply:
Emm. I'm not sure about this. I consider autofix is just a robot that executing `just fix`  for you. But no matter who  are executing `just fix` we still need to validate the fix.